EIPS/eip-1803.md

```diff
@@ -0,0 +1,41 @@
+---
+eip: 1803
+title: Rename opcodes for clarity
+author: Alex Beregszaszi (@axic)
+type: Standards Track
+category: Interface
+layer: Applications
```

Comment:
I don't think this field is supposed to be in the header field.

Reply:
Copied it from https://github.com/ethereum/EIPs/blob/master/EIPS/eip-6.md, but it seems since that was merged it is obsolete. Will remove.

Reply:
Done.